kyo-core/shared/src/main/scala/kyo/Fiber.scala

```diff
@@ -49,7 +49,7 @@ object Fiber extends FiberPlatformSpecific:
       * @return
       *   A Fiber that completes successfully with the given value
       */
-    def success[E, A](v: A): Fiber[E, A] = result(Result.success(v))
+    def success[E, A](v: A)(using Frame): Fiber[E, A] < IO = result(Result.success(v))
```

Comment:
These methods were missing the `IO` suspension since they initialize objects with mutable state.

Reply:
Actually, I've just realized that the returned values are effectively final. `IOPromise` can't change its state after it's complete, even in case of interruption. I'll remove the suspension.